callisto/notification/api.py

```diff
@@ -57,8 +67,8 @@ def send_matching_report_to_school(cls, matches, identifier):
         cls.send_email_to_coordinator(pdf, 'match_delivery', report_id)
 
     @classmethod
-    def send_user_notification(cls, form, notification_name):
-        notification = cls.model.objects.on_site().get(name=notification_name)
+    def send_user_notification(cls, form, notification_name, site_id=None):
+        notification = cls.model.objects.on_site(site_id).get(name=notification_name)
```

Comment:
django checks for a `settings.SITE_ID`, and raises an exception if it doesn't find one

https://github.com/SexualHealthInnovations/callisto-core/blob/5b27a77a00f34693e02822fc755880e077208012/callisto/notification/managers.py#L8

https://github.com/django/django/blob/master/django/contrib/sites/models.py#L50